ietf-layer0-types-ext.yang

```diff
@@ -684,10 +766,10 @@ module ietf-layer0-types-ext {
       uses cd-pmd-penalty ;
     }
     leaf max-diff-group-delay  {
-           type int32;
-           config false;
-           description "Maximum Differential group delay of this mode
-                        for this lane";
+      type int32;

```

Comment:
shouldn't this be a decimal value ?
unit is missing

Reply:
To be discussed with other co-authors in the upcoming weekly call (after the I-D submission): see issue #39 

Co-Authored-By: sergio belotti [sergio.belotti@nokia.com](mailto:sergio.belotti@nokia.com)